src/ui/public/vis/editors/default/agg_params.html

```diff
@@ -1,6 +1,9 @@
 <div ng-if="aggIsTooLow" class="form-group">
-  <p class="visEditorAggParam__error">
-    "{{ agg.schema.title }}" aggs must run before all other buckets!
+  <p class="visEditorAggParam__error"
+    i18n-id="common.ui.vis.editors.aggParams.errors.aggWrongRunOrderErrorMessage"
+    i18n-default-message="{schema} aggs must run before all other buckets!"
+    i18n-values="{ schema: '&quot;' + agg.schema.title + '&quot;' }"
```

Comment:
As of some earlier discussions around quoting, I think we decided to only keep quotes as part of the parameter value, if they are actually belonging to it (e.g. because we are representing a JSON value), in this case it's just regular "quotes" to highlight something within a sentence, which could be [different per language](https://en.wikipedia.org/wiki/Quotation_mark#Summary_table) and thus should be part of the default-message instead.